src/components/report/FeatureControls.js

```diff
@@ -49,8 +51,16 @@ export default {
 				if (state === null) continue;
 				styles += `${glue} "${feature.tag}" ${state ? "1" : "0"}`;
 				glue = ",";
+				// Poor man's code formatting
+				if (++counter % maxProps === 0) {
+					glue = `,\n                      `;
+				}
+			}
+			if (styles) {
+				return `font-feature-settings:${styles};`;
```

Comment:
Could this be moved to the engine perhaps? (Could be a separate PR)

Reply:
Perhaps, but the engine currently looks only at the font contents and returns generic CSS for that. This specific code here is dynamic: it needs to change in response to the user playing around with on/off/indeterminate buttons.

So for now I feel it belongs here, but maybe when improving the engine's CSS generation, we can account for this kind of dynamic CSS creation and move it.